widgets/class-monk-language-switcher.php

```diff
@@ -46,14 +46,23 @@ public function widget( $args, $instance ) {
 		$title                = ! empty( $instance['title'] ) ? $instance['title'] : __( 'Languages', 'monk' );
 		$flag                 = ! empty( $instance['flag'] ) ? true : false;
 		$active_languages     = get_option( 'monk_active_languages' );
-		$current_language     = isset( $_GET['lang'] ) ? sanitize_text_field( wp_unslash( $_GET['lang'] ) ) : get_option( 'monk_default_language' );
+		$current_language     = get_query_var( 'lang' ) ? sanitize_text_field( wp_unslash( get_query_var( 'lang' ) ) ) : get_option( 'monk_default_language' );
```

Comment:
I think you can remove `wp_unslash` from there.
We also need a default value like `get_option( 'monk_default_language', false );`